treeherder/client/thclient/client.py

```diff
@@ -552,6 +552,25 @@ def validate(self):
         for d in self.data:
             d.validate()
 
+    def get_chunks(self, chunk_size):
+        """
+        Return a list of new collections broken into chunks of size ``chunk_size``.
+
+        Each chunk will be a ``TreeherderCollection`` of the same
+        type as the original with a max of ``chunk_size`` count of
+        ``TreeherderData`` objects.
+
+        Each collection must then be POSTed individually.
+        """
+        artifact_collections = []
```

Comment:
I missed this on the first review iteration, can you please rename this to something more generic? Like collection_list for example, or treeherder_collections.